### PR TITLE
Move CI from CircleCI to Drone

### PR DESCRIPTION
**Pull request by sirikid**

### CODE REVIEW

File: .credo.exs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        {Credo.Check.Consistency.SpaceInParentheses, []},
+        {Credo.Check.Consistency.TabsOrSpaces, []},
+
+        # Design Checks
+        {Credo.Check.Design.AliasUsage, [priority: :low, if_nested_deeper_than: 2, if_called_more_often_than: 0]},
+        {Credo.Check.Design.TagTODO, [exit_status: 2]},
+        {Credo.Check.Design.TagFIXME, []},
+
+        # Readability Checks
+        {Credo.Check.Readability.AliasOrder, []},
```
Comment (hissssst) [inline .credo.exs]: I hate this check. Remove it maybe?

**Review by hissssst — CHANGES_REQUESTED** [review]

File: .credo.exs (hunk 1 above)
Comment (jrogov) [reply .credo.exs]: What's the usecase you see where this check might be a handicap?

**Comment by jrogov** [comment]
Added some Nicene checks. Thoughts on this?

File: .credo.exs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        # Following might be helpful, but is invisible to clients:
+        # consider in future
+        {Nicene.AvoidImportsFromCurrentApplication, false},
+        {Nicene.ConsistentFunctionDefinitions, false},
+        {Nicene.DocumentGraphqlSchema, false},
+        {Nicene.EctoSchemaDirectories, false},
+        {Nicene.FileAndModuleName, []},
+        {Nicene.FileTopToBottom, []},
+        {Nicene.NoSpecsPrivateFunctions, false},
+        {Nicene.PublicFunctionsFirst, []},
```
Comment (sirikid) [inline .credo.exs:117]: What is this for?

Comment (jrogov) [reply .credo.exs:117]: From the docs:
```
defmodule Nicene.FileAndModuleName do
  @moduledoc """
  Check to ensure that module names correspond to the file that it is defined it.
...
defmodule Nicene.FileTopToBottom do
  @moduledoc """
  Invokations of functions should be before definitions of those functions.
...
defmodule Nicene.NoSpecsPrivateFunctions do
  @moduledoc """
  Private functions should not have typespecs
...
defmodule Nicene.PublicFunctionsFirst do
  @moduledoc """
  Public functions in a module should be defined before private functions.
```

Comment (sirikid) [reply .credo.exs:117]: Nevermind, let's go for it.

Comment (jrogov) [reply .credo.exs:117]: The idea is to try these additions out, and drop them if they only block development. 
Now I'm not sure if this project is alive at all, given there're quite a lot of PRs now and it has quite strange bugs here and there (e.g. `PublicFunctionsFirst` doesn't take nested modules or module defs inside __using__ macro, and therefore gives false positives)

**Review by jrogov — COMMENTED** [review]
LGTM

**Review by jrogov — COMMENTED** [review]
LGTM

**Review by hissssst — APPROVED** [review]

**Pull request merged**